src/pages/search-results.js

```diff
@@ -0,0 +1,272 @@
+import React, { useEffect, useState } from 'react';
+import PropTypes from 'prop-types';
+import { css } from '@emotion/react';
+import styled from '@emotion/styled';
+import {
+  Icon,
+  Button,
+  Link,
+  search,
+  Spinner,
+  Surface,
+  useLocale,
+  useQueryParams,
+} from '@newrelic/gatsby-theme-newrelic';
+
+import { usePagination, DOTS } from '../hooks/usePagination';
+
+const SearchResultPageView = ({ pageContext }) => {
+  const { queryParams } = useQueryParams();
+  const query = queryParams.get('query');
```

Comment:
we should handle the case where `query` is `undefined` or an empty string. i think navigating to the home page would be fine